ReactAndroid/src/androidTest/js/UIManagerTestModule.js

```diff
@@ -9,214 +9,248 @@
-  displayName: 'AbsolutePositionBottomRightTestApp',
-  _styles: StyleSheet.create({
-    container: {
-      width: 100,
-      height: 100,
-    },
-    absolute: {
-      position: 'absolute',
-      bottom: 15,
-      right: 10,
-      width: 50,
-      height: 60,
-    },
-  }),
-  render: function() {
+type AbsolutePositionBottomRightTestAppProps = $ReadOnly<{||}>;
+class AbsolutePositionBottomRightTestApp extends React.Component<
+  AbsolutePositionBottomRightTestAppProps,
+> {
+  render() {
     return (
       <View
-        style={this._styles.container}
+        style={AbsolutePositionBottomRightTestAppStyles.container}
         testID="container"
         collapsable={false}>
-        <View style={this._styles.absolute} collapsable={false} />
+        <View
+          style={AbsolutePositionBottomRightTestAppStyles.absolute}
+          collapsable={false}
+        />
       </View>
     );
+  }
+}
+
+const AbsolutePositionBottomRightTestAppStyles = StyleSheet.create({
+  container: {
+    width: 100,
+    height: 100,
+  },
+  absolute: {
+    position: 'absolute',
+    bottom: 15,
+    right: 10,
+    width: 50,
+    height: 60,
   },
 });
 
-var CenteredTextView = createReactClass({
-  displayName: 'CenteredTextView',
-  _styles: StyleSheet.create({
-    parent: {
-      width: 200,
-      height: 100,
-      backgroundColor: '#aa3311',
-      justifyContent: 'center',
-      alignItems: 'center',
-    },
-    text: {
-      fontSize: 15,
-      color: '#672831',
-    },
-  }),
-  render: function() {
+type CenteredTextViewProps = $ReadOnly<{|
+  text?: ?string,
+|}>;
+class CenteredTextView extends React.Component<CenteredTextViewProps> {
+  render() {
     return (
       <View collapsable={false}>
-        <View style={this._styles.parent} collapsable={false}>
-          <Text style={this._styles.text} testID="text">
+        <View style={CenteredTextViewStyles.parent} collapsable={false}>
+          <Text style={CenteredTextViewStyles.text} testID="text">
             {this.props.text}
           </Text>
         </View>
       </View>
     );
+  }
+}
+
+const CenteredTextViewStyles = StyleSheet.create({
+  parent: {
+    width: 200,
+    height: 100,
+    backgroundColor: '#aa3311',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  text: {
+    fontSize: 15,
+    color: '#672831',
   },
 });
 
-var flushUpdatePositionInList = null;
-var UpdatePositionInListTestApp = createReactClass({
-  displayName: 'UpdatePositionInListTestApp',
-  _styles: StyleSheet.create({
-    element: {
-      height: 10,
-    },
-    active: {
-      height: 50,
-    },
-  }),
-  getInitialState: function() {
-    flushUpdatePositionInList = () => this.setState({active: true});
-    return {active: false};
-  },
-  render: function() {
+let flushUpdatePositionInList = null;
+type UpdatePositionInListTestAppProps = $ReadOnly<{||}>;
+type UpdatePositionInListTestAppState = $ReadOnly<{|
+  active?: ?boolean,
```

Comment:
Are you sure this is nullable? It looks like it defaults to false and is only ever set to true.

Reply:
Heh, same comment as @RSNara